src/components/login/Registration.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import "./Login.css";
+
+class Registration extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      name: "",
+      surname: "",
+      username: "",
+      password: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event){
```

Comment:
Codacy found an issue: [Method 'handleChange' has a complexity of 5. Maximum allowed is 4.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6356901)